apps/pie-docs/src/src.11tydata.js

```diff
@@ -0,0 +1,26 @@
+const projectSettings = require('./_data/projectSettings');
+
+// Only build drafts when site is in development mode
+const shouldBuildDrafts = projectSettings().environment === 'development';
+
+module.exports = {
+    eleventyComputed: {
+        permalink (data) {
+            // If the page is in `draft:true` and the site is not in 'development' mode, do not build page.
+            if (data.draft && !shouldBuildDrafts) {
+                return false;
+            }
+            // Return the original value (which could be `false`, or a custom value,
+            // or default empty string).
+            return data.permalink;
+        },
+        eleventyExcludeFromCollections (data) {
```

Comment:
I understand the point of this, and it's probably nice from a future proof perspective. But do we actually use collections atm? I know we use collections.all, but as far as I'm aware `eleventyExcludeFromCollections` doesn't exclude from `all` (I could be mistaken)

Reply:
👍